<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>520</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>动画设置</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>动画</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>倍数</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Factor">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Jump">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>260</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>跳转速度(mm/s)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>300</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>标记速度(mm/s)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Mark">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>300</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>50</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>260</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>平台宽度W(mm)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_platform_W">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>260</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>200</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>300</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>平台高度W(mm)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_platform_H">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>300</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>200</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_platform_X">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>220</y>
     <width>40</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>150</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>220</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>平台位置(X,Y)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_platform_Y">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>220</y>
     <width>40</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>150</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
